### PR TITLE
More detailed build guide

### PR DESCRIPTION
**Pull request by quinton-ashley**
Added more detail to the build guide and split it up into sections.

### CODE REVIEW

File: box2d3-wasm/README.md [hunk 1]
```diff
@@ -1,20 +1,67 @@
-# Box2d3-wasm
+# box2d3-wasm
 
-Let's try and use box2d v3 on the web.
+Let's try and use [Box2D v3][] on the web via [wasm][]!
```
Comment (Birch-san) [inline box2d3-wasm/README.md:3]: the URLs(?) here look unfinished

Comment (Birch-san) [reply box2d3-wasm/README.md:3]: oh, I see, they get moved down to the bottom of the page…

File: box2d3-wasm/README.md [hunk 2]
```diff
@@ -1,20 +1,67 @@
...
 emmake make -j8 -C cmake-build
 FLAVOUR=simd TARGET_TYPE=Debug ./shell/1_build_wasm.sh
+```
+
+## Run Demos
+
+Install the "local-web-server" package globally (or locally in the `demo` directory).
+
+```bash
+npm i local-web-server -g
```
Comment (Birch-san) [inline box2d3-wasm/README.md]: there's no need to install local-web-server globally; it's already a devDependency of demo, so you can just install demo's dependencies. you're gonna need to yarn install the demo either way to get the npm link to happen

File: box2d3-wasm/README.md (hunk 1 above)
Comment (quinton-ashley) [reply box2d3-wasm/README.md:3]: yeah some fancy markdown there haha, I could just link them up top if you want

File: box2d3-wasm/README.md (hunk 2 above)
Comment (quinton-ashley) [reply box2d3-wasm/README.md]: ah okay, so yarn is necessary then. I can edit that.

Comment (quinton-ashley) [reply box2d3-wasm/README.md]: just fixed this

**Comment by Birch-san** [comment]
btw I've now published a release build to:  
https://www.npmjs.com/package/box2d3-wasm

**Comment by quinton-ashley** [comment]
awesome!!

But if that's going to be the README for the npm package the first h2 should probably be titled "Build Pre-Requisites"

**Comment by Birch-san** [comment]
updated readme, published patch release

**Pull request merged**